src/testRunner/unittests/tsserver/getEditsForRefactor.ts

```diff
@@ -0,0 +1,53 @@
+import * as ts from "../../_namespaces/ts.js";
+import {
+    openFilesForSession,
+    TestSession,
+} from "../helpers/tsserver.js";
+import {
+    createServerHost,
+    File,
+} from "../helpers/virtualFileSystemWithWatch.js";
+
+describe("unittests:: tsserver:: getEditsForRefactor", () => {

```

Comment:
Please move this test case to tests/case/fourslash instead - fourslash has method for "moveToFile" that verifies that this is correct.

Reply:
@sheetalkamat Thanks! I had overlooked them before thinking they were something else. Have now replaced my first attempt with a fourslash version.